radlab-ui/webapp/src/utils/terraform.ts

```diff
@@ -113,7 +113,11 @@ const mapHclToUIVar = (
       ? false
       : null,
     // In TF, desription = "" is how we say it's optional
-    required: hclVar.default ? hclVar.default !== "" : false,
+    required: hclVar.default
+      ? hclVar.default !== ""
+      : hclVar.default === undefined
+      ? true
+      : false,
```

Comment:
Can we simplify to `required: typeof hclVar.default === "undefined" || hclVar.default === ""`